deployment/ansible/group_vars/development

```diff
@@ -18,6 +18,8 @@ graphite_host: "{{ services_ip }}"
 statsite_host: "{{ services_ip }}"
```

Comment:
Can you apply this change to your PR? https://github.com/hectcastro/model-my-watershed/commit/e00c02d32c81be418408563e2844de4a91e6412e